styles.css

```diff
@@ -1 +1,83 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
```

Comment:
Wat mij opvalt aan je styles is dat het er allemaal keurig uitziet. je hebt alleen ook 2 a 3 verschillende manieren om je kleuren te benoemen. misschien dat je alles in rgb of hsl kan doen? zodat alles het zelfde is? Tip: ik heb hier een color converter voor je zodat je alles het zelfde kan doen op je eigen manier: https://www.w3schools.com/colors/colors_converter.asp

Reply:
Heb je gelijk in! Ik was me niet bewust van het feit dat ik verschillende kleur codes gebruikt had. Ik dacht, geef het maar een kleurtje en klaar. Maar ziet er inderdaad wel netter uit om alles dezelfde kleurtype te gebruiken! Thanks voor de link